### DISCUSSION TITLE
Asking For Documentation

### DISCUSSION THREAD

**Discussion by kRHYME7** [prasanthrangan/hyprdots#767] [Ideas]
Hello everyone, this repo has 3k+ stars now. WOW! 
I hope someone here is interested in contributing some documentation/WIKI. We have a lot of hidden gem features undocumented. This would lead to more redundant issues.

**Comment by ghost** [comment]
I'd be interested in working on improving documentation, where do you think it'd be most useful to work on? I'd probably want to start listing out each individual features people might not know about, maybe showing what people can do with the dots too would be nice.

**Reply by kRHYME7** [reply]
Maybe we can start with those Q/A and some issues that were almost duplicated.
